deepcell/applications/cell_tracking_test.py

```diff
@@ -66,7 +66,7 @@ class TestCellTracking(test.TestCase):
     def test_cell_tracking_app(self):
         with self.cached_session():
             # test instantiation
-            app = CellTracking(use_pretrained_weights=False)
+            app = CellTracking()
```

Comment:
I suggest creating a dummy model here and passing it to `CellTracking`. That way the tests won't have to download anything from S3 and will prevent failures due to network errors or service outages.

Reply:
Done.